editors/vscode/package.json

```diff
@@ -144,7 +149,77 @@
+                        "media": {
+                            "svg": "static/assets/rust-logo.svg",
+                            "altText": "rust logo"
+                        }
+                    },
+                    {
+                        "id": "create_slint_file",
+                        "title": "Create a .slint File",
+                        "description": "[Open](command:toSide:workbench.action.files.openFile) or [create](command:toSide:workbench.action.files.newUntitledFile?%7B%22languageId%22%3A%22slint%22%7D) a `.slint` file. Be sure to save it with the \".slint\" extension, such as \"helloworld.slint\". \n[Create a .slint File](command:toSide:workbench.action.files.newUntitledFile?%7B%22languageId%22%3A%22slint%22%7D)",
+                        "media": {
+                            "markdown": "static/walkthroughs/welcome/create_slint_file.md"
+                        }
+                    },
+                    {
+                        "id": "live_preview",
+                        "title": "Live Preview your UI",
+                        "description": "Click on $(play) Show Preview above any component in your editor showing a `.slint` file to a preview of your user interface. It updates as you type!",
+                        "media": {
+                            "image": "static/walkthroughs/welcome/show_preview.png",
+                            "altText": "live preview in action"
+                        },
+                        "completionEvents": [
+                            "onCommand:slint.showPreview"
+                        ]
+                    },
+                    {
+                        "id": "property_editor",
+                        "title": "Edit Properties",
+                        "description": "When your cursor is on an Slint Element in your editor, the Property Editor will show all possible properties defined on the Element.\n[Open Properties Editor](command:slint.showPropertiesEditor)",
+                        "media": {
+                            "markdown": "static/walkthroughs/welcome/properties_editor.md"
+                        },
+                        "completionEvents": [
+                            "onView:slint.propertiesView"
+                        ]
+                    },
+                    {
+                        "id": "welcome_page",
+                        "title": "Find Help on the Welcome Page",
+                        "description": "Use the Slint Welcome Page to find documentation and further links\n[Open Welcome Page](command:slint.showWelcome)",
+                        "media": {
+                            "image": "static/walkthroughs/welcome/welcome.png",
+                            "altText": "The Welcome Page"
+                        },
+                        "completionEvents": [
+                            "onView:slint.showWelcome"
+                        ]
+                    }
```

Comment:
IMHO this walk-through should replace the welcome page.
We can have a step to visit the docs, and link to the documentation in the markdown page.